Automattic/jetpack
# Unified payments block selector

Shows a menu of payment blocks after inserting this one

See paYJgx-1Hh-p2
Fixes 599-gh-Automattic/dotcom-manage

Still plenty to do:
 - [x] Saving isn't implemented
 - [x] The blocks that can be added by this will get duplicated in the block
   selector, as they still exist in standalone mode.
 -  [x] Inner block titles etc are just placeholder content
 - [x] Icons for the inner blocks need adding
  - [x] Icons for the inner blocks need to be the same size
  - [x] Make the selected inner block replace the one-payment block when selected, rather than just being contained
 - [x] Wrapper block needs final icon/title/description and other wording







#### Jetpack product discussion




#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






##### Setup
Jetpack env:
* Run pnpm build-extensions from projects/plugins/jetpack/extensions

wpcom:
 * Patch sandbox as usual

Atomic:
 * Use jetpack beta plugin to select this branch


##### Test
* Load gutenberg
* In the block inserter `/` you should now see 'Payments button', 'Donations form' and 'Payments'
* Try inserting a 'Payment' block, you'll then be given a choice of payments button or donations form
